File: benchmark/corpus/04_off_by_one/e_pairs.py
# Compares each item with the next one; the last item has no next.


def rises(xs):
    count = 0
    i = 0
    while i < len(xs):
        if xs[i + 1] > xs[i]:  # DANGER
            count += 1
        i += 1
    return count


print(rises([2500, 45000, 12000]))

File: benchmark/corpus/04_off_by_one/e_pairs.js
// Compares each item with the next one; the last item has no next.
function rises(xs) {
  let count = 0;
  let i = 0;
  while (i < xs.length) {
    if (xs[i + 1] > xs[i]) count++; // DANGER
    i++;
  }
  return count;
}

console.log(rises([2500, 45000, 12000]));
